src/ZfcRbac/Factory/ControllerGuardFactory.php

```diff
@@ -29,7 +28,7 @@
  * @author  Michaël Gallego <mic.gallego@gmail.com>
  * @license MIT
  */
-class ControllerGuardFactory implements FactoryInterface, MutableCreationOptionsInterface
```

Comment:
removing `MutableCreationOptionsInterface` please look how servicemanager v2 keep compatibility: https://github.com/zendframework/zend-servicemanager/blob/release-2.7/src/AbstractPluginManager.php#L322-L328

basicaly instead of `MutableCreationOptionsInterface` it checks for `setCreationOptions()` not sure it works without it. Possible BC break unless this is handles somewhere else but don't know if this is good idea.